seed/seed20.go

```diff
@@ -762,3 +764,30 @@ func (s *seed20) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed20) LoadAutoImportAssertion(commitTo func(*asserts.Batch) error) error {
+	if s.model.Grade() == asserts.ModelDangerous {
+		autoImportAssert := filepath.Join(s.systemDir, "auto-import.assert")
+		if _, err := os.Stat(autoImportAssert); err == nil {
+			af, err := os.Open(autoImportAssert)
+			if err != nil {
+				logger.Noticef("failed to open auto-import assert(%s): %v\n", autoImportAssert, err)
+				return nil
+			}
+			defer af.Close()
+			var ar io.Reader
+			ar = af
```

Comment:
ehm :-) `af` ->`assertionFile`

I don't think you need a new variable for the reader here, just passing the file to `AddStream` should work.

Reply:
changed